src/Parser.js

```diff
@@ -315,15 +315,21 @@ export default class Parser {
 
             if (lex.text === "\\limits" || lex.text === "\\nolimits") {
                 // We got a limit control
-                const opNode = checkNodeType(base, "op");
+                let opNode = checkNodeType(base, "op");
                 if (opNode) {
                     const limits = lex.text === "\\limits";
                     opNode.limits = limits;
                     opNode.alwaysHandleSupSub = true;
                 } else {
-                    throw new ParseError(
-                        "Limit controls must follow a math operator",
-                        lex);
+                    opNode = checkNodeType(base, "operatorname");
+                    if (opNode && opNode.alwaysHandleSupSub) {
+                        const limits = lex.text === "\\limits";
+                        opNode.limits = limits;
```

Comment:
> So limits aren't allowed for `\operatorname`

It might be more precise to say that `\limits` is ineffective when applied to `\operatorname`.  You can write a `\limits` after an `\operatorname` without a log error, but the sub will only be shifted below in the case of `\operatorname*`.